lib/tasks/search.rake

```diff
@@ -4,6 +4,21 @@ namespace :search do
     Document.find_by(content_id: args[:content_id]).published_edition.update_in_search_index
   end
 
+  desc "Re-index a collection of Documents with specified world location. Takes a `world_location_slug` as argument."
+  task :resend_document, [:world_location_slug] => [:environment] do |_, args|
```

Comment:
```suggestion
  task :resend_documents_in_world_location, [:world_location_slug] => [:environment] do |_, args|
```

I think this might be clearer for what the task does.